src/org/usfirst/frc/team467/robot/Grabber.java

```diff
@@ -27,14 +30,20 @@
 	private boolean hadCube = false;
 	private boolean hasCube = false;
 	private OpticalSensor os;
+	private GrabberSolenoid rightGrab;
+	private GrabberSolenoid leftGrab;
+	
+	//private boolean grabberButtonDown = false;
 
 	private Grabber() {
-		if (RobotMap.HAS_GRABBER && !RobotMap.useSimulator) {
+		if (RobotMap.HAS_GRABBER && !RobotMap.useSimulator && RobotMap.RIGHT_GRABBER_SOLENOID_EXISTS) {
```

Comment:
I suggest writing it such that if either of the grabber solenoid or wheels doesn't work, the other still will, instead of killing all grabber functionality if one of the pieces fails.